docs/install_spock.md

```diff
@@ -187,10 +187,6 @@ Use this GUC to specify which operations/transactions are written to the excepti
 * `discard` - Add a row to the `spock.exception_log` table for any discarded operation; successful transactions are not logged.
 * `none` - Instructs the server to not log any operation or transactions to the exception log table.
 
-#### `spock.exception_replay_queue_size`
```

Comment:
@susan-pgedge, do you think we should keep that here and mention that it's deprecated?